### ISSUE TITLE
nco and cdo errors

### ISSUE THREAD

**Issue by DamienIrving** [ioos/conda-recipes#776]
I've installed [nco](http://anaconda.org/ioos/nco) and [cdo](http://anaconda.org/ioos/cdo) on OS X (version 10.11) using the relevant IOOS packages but then get the following errors when I try and run them from the command line:
```
$ ncks
dyld: Library not loaded: @rpath/libgssapi_krb5.2.2.dylib
  Referenced from: /Users/irv033/miniconda2/envs/IOOS/bin/ncks
  Reason: image not found
Trace/BPT trap: 5

$ cdo
dyld: Library not loaded: /usr/local/lib/libuuid.16.dylib
  Referenced from: /Users/irv033/miniconda2/envs/IOOS/bin/cdo
  Reason: image not found
Trace/BPT trap: 5
```
(It looks like this issue has been discussed in the past [here](https://github.com/Anaconda-Server/docs.anaconda.org/issues/98), however @CloLanglais informs me that she isn't sure if that solution works with the latest version of OS X)

What would be the best solution to this problem and would it be possible to update http://anaconda.org/ioos/nco and http://anaconda.org/ioos/cdo accordingly?

**Comment by kwilcox** [comment]
I don't have OS X to test with. Can you try `conda install krb5` (install from the default channel) and see if that clears up the `ncks` issue?

**Comment by DamienIrving** [comment]
@kwilcox Yep, `conda install krb5` fixes the issue with any/all of the nco operators (like `ncks`). Thanks!

It doesn't fix the problem with cdo though...

**Comment by ocefpaf** [comment]
Looks like `nco` is suffering from the same problem as #780. PR inbound. I am working on a solution for `cdo` too. Looks like I'll have to package `libuuid`.

I wonder if changing from the CIs gcc to conda's gcc is the root of the problem...

**Comment by ocefpaf** [comment]
Fixed in #797 and #796. Let's re-open if the problem re-surfaces, but the CIs are green.
